ArchiSteamFarm/NLog/ArchiLogger.cs

```diff
@@ -115,7 +115,7 @@ public sealed class ArchiLogger {
 	}
 
 	[PublicAPI]
-	public void LogNullError(string nullObjectName, [CallerMemberName] string? previousMethodName = null) {
+	public void LogNullError(object? nullObject, [CallerArgumentExpression("nullObject")] string? nullObjectName = null, [CallerMemberName] string? previousMethodName = null) {
```

Comment:
```suggestion
	public void LogNullError(object? nullObject, [CallerArgumentExpression(nameof(nullObject))] string? nullObjectName = null, [CallerMemberName] string? previousMethodName = null) {
```

I didn't test this, give it a try.

Reply:
No need to try. I already have some weeks ago. It won't compile.